composer.json

```diff
@@ -17,7 +17,6 @@
     "require": {
         "php": ">=5.5.0",
         "illuminate/console": "~5.1.10|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*",
-        "illuminate/log": "~5.1.10|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*",
```

Comment:
Don't we still need this, just so that we have a package that implements that LoggerInterface? Or does it not matter?